nims-notes-things/src/redux/reducers/noteReducer.js

```diff
@@ -1,16 +1,20 @@
+import { setSorted } from "../actions/noteActions";
 import * as actions from "../actions/noteActionTypes";
 
 export const initialState = {
-  value: [], //whatever passed in - state arrary -> passed inside export function
+  //whatever passed in - state arrary -> passed inside export function
+  value: [],
```

Comment:
seed to add sorted to this:
```
export const initialState = {
   sorted: false,
   value: []
}
```
`value` is and should only be used for the notes handling